engine/engine.go

```diff
@@ -151,6 +152,8 @@ func flagSet() *pflag.FlagSet {
 	flags.String(pkg.ConfActingPartyCN, defs.ActingPartyCn, "The acting party Common name used in contracts")
 	flags.Bool(irma.ConfSkipAutoUpdateIrmaSchemas, defs.SkipAutoUpdateIrmaSchemas, "set if you want to skip the auto download of the irma schemas every 60 minutes.")
 	flags.Bool(pkg.ConfEnableCORS, defs.EnableCORS, "Set if you want to allow CORS requests. This is useful when you want browsers to directly communicate with the nuts node.")
+	flags.Bool(oauth.ConfGenerateOAuthKeys, defs.GenerateOAuthKeys, "Auto generate OAuth JWT signing key if missing.")
+	flags.String(oauth.ConfOAuthSigningKey, defs.OAuthSigningKey, "Path to PEM encoded private key.")
```

Comment:
I had to read the code to realise the authorization server isn't enabled when this is empty, maybe add that to the config option explanation.